docs/detections/rules-ui-manage.asciidoc

```diff
@@ -60,20 +70,34 @@ image::images/install-prebuilt-settings.png[]
 image::images/install-prebuilt-rules.png[]
 
 [float]
-[[manage-rules-ui]]
-=== Modify existing rules
+[[edit-rules-settings]]
+=== Modify existing rules settings
+
+You can edit the settings of an existing rule. You can also select multiple rules, then edit certain settings in bulk (index patterns and tags).
 
-You can clone, edit, activate, deactivate, and delete rules:
+NOTE: For prebuilt Elastic rules, you can't modify most settings. You can only edit <<rule-schedule, rule actions>> and <<detections-ui-exceptions, add exceptions>>.
 
 . Go to *Detect* -> *Rules*.
 . Do one of the following:
-* Click the *All actions* icon (*...*) on the appropriate row, then select the required action.
-* In the *Rule* column, select all the rules you want to modify, then select the
-required action from the *Bulk actions* menu.
-. To activate or deactivate a rule, click the *Activated* toggle button.
+* Edit a single rule: Click the *All actions* icon (*...*) on a rule, then select *Edit rule settings*. The *Edit rule settings* view opens, where you can modify the <<rules-ui-create, rule's settings>>.
+* Edit multiple rules: Select all the rules you want to edit, then select an action from the *Bulk actions* menu:
+** *Index patterns*: Add or delete the index patterns used by all selected rules.
+** *Tags*: Add or delete tags on all selected rules.
+. On the flyout that opens, enter the settings to change on the rules.
+. If available, select *Overwrite all selected _x_* to overwrite the settings on the rules. For example, if you're adding tags to multiple rules, selecting *Overwrite all selected rules tags* removes all the rules' original tags and replaces them with the tags you specify.
+. Click *Save*.
 
-NOTE: For prebuilt rules, you can only activate, deactivate, delete, edit
-<<rule-schedule, rule actions>>, and <<detections-ui-exceptions, add exceptions>>.
+[float]
+[[manage-rules-ui]]
+=== Manage rules
+
+You can duplicate, enable, disable, and delete rules:
+
+. Go to *Detect* -> *Rules*.
+. Do one of the following:
+* Click the *All actions* icon (*...*) on a rule, then select the required action.
```

Comment:
```suggestion
* Click the *All actions* icon (*...*) on a rule, then select the desired action.
```

Reply:
just putting it out there — to me required sounds like we are requiring them to do it. Probably not confusing in this context but I still like desire 🤷

Reply:
@benironside I'm not a fan of "desire" in technical docs, but definitely agree that "required" isn't right. I think we can omit an adjective altogether :
```suggestion
* Click the *All actions* icon (*...*) on a rule, then select an action.
```